addons/binding/org.openhab.binding.wso2iots/src/main/java/org/openhab/binding/wso2iots/internal/jsonTemp/Values.java

```diff
@@ -15,7 +15,7 @@
  * The {@link Values} class returns the temperature value
  * from json wso2iotsresponse string
  *
- * @author wso2_Ramesha - Initial contribution
+ * @author Ramesha Karunasena - Initial contribution
  */
 
 public class Values {
```

Comment:
The previous comment remains valid - I would suggest combining all json classes into single class, because it leads to a much simpler code which is easier to understand and maintain. Unless there are some reasons not to :)